contracts/ops/chain-id.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
+set -eu
+set -o pipefail
```

Comment:
nit: we can do this in one line `set -euo pipefail`

Reply:
that is cool! I did not know that! next time 😉